docker/Dockerfile-gbpserver

```diff
@@ -1,5 +1,13 @@
 FROM registry.access.redhat.com/ubi8/ubi:latest
 COPY dist-static/gbpserver /usr/local/bin/
-
+# Required OpenShift Labels
+LABEL name="ACI CNI gbserver" \
```

Comment:
Pls change "gbserver" to "gbpserver" everywhere in this file.

Reply:
sure...